runtime-testsuite/pom.xml

```diff
@@ -119,6 +119,8 @@
                     <argument>${basedir}/test</argument>
                     <argument>-templates</argument>
                     <argument>${basedir}/resources/org/antlr/v4/test/runtime/templates</argument>
+                    <argument>-target</argument> <!-- TODO remove this before merging with upstream! -->
```

Comment:
Since there are already a lot of TODOs, it might be easier for us to find stuff we need to clean up by using a string we need to remove anyway, like "pboyer", since that's already hard-coded in various import paths right now for convenience. Maybe something like `PBOYER TODO: ...`?